docs/literate/src/files/adding_parabolic_terms.jl

```diff
@@ -0,0 +1,88 @@
+#src # Adding parabolic terms (advection-diffusion).
```

Comment:
Did you check the output?

Reply:
Yup; looks like the following
<img width="907" alt="Screen Shot 2022-08-25 at 6 58 36 AM" src="https://user-images.githubusercontent.com/1156048/186658424-8b65366d-035a-46ca-818a-5a7c2bf44990.png">